scripts/ci/release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dir=$(dirname $0)
+projectRoot=${dir}/../..
+
+if [ -z "$DOCKER_NAMESPACE" ]; then
+  DOCKER_NAMESPACE="gcr.io/$(gcloud info \
```

Comment:
I don't think it's necessary to use maven properties to set the DOCKER_NAMESPACE in this build config. Cloud container builder already has its own ["substitutions" mechanism](https://cloud.google.com/container-builder/docs/how-to/build-requests) for doing this. 

If you think it's important to be able to override the DOCKER_NAMESPACE from this script, the code that invokes the build can handle this substitution, like this: 

```
DOCKER_NAMESPACE=gcr.io/$something
gcloud container builds submit . --config=build.yaml --substitutions=_DOCKER_NAMESPACE=$DOCKER_NAMESPACE
```